src/Backend/OnboardingInsuranceAPI/Areas/User/RegisterUser.cs

```diff
@@ -0,0 +1,60 @@
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using OnboardingInsuranceAPI.ErrorHandling;
+using OnboardingInsuranceAPI.Extensions;
+using OnboardingInsuranceAPI.Services;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace OnboardingInsuranceAPI.Areas.User;
+
+public class RegisterUser : IHandler
+{
+    private readonly DataContext _context;
+    private readonly ILogger<RegisterUser> _logger;
+
+    public RegisterUser(DataContext context, ILogger<RegisterUser> logger)
+    {
+        _context = context;
+        _logger = logger;
+    }
+
+    public async Task Handle(string pid, string email, HttpRequestData req)
```

Comment:
NO! We discussed this. Handler shouldn't care about HTTP! read the pid before hand.